packages/desktop-app/electron-builder.json

```
@@ -16,5 +16,12 @@
     "artifactName": "${productName} Installer ${version}.${ext}",
     "license": "LICENSE.md",
     "oneClick": false
+  },
+  "linux": {
+    "target": ["deb", "rpm", "snap"],
```

Let's also include the AppImage target:

```suggestion
    "target": ["AppImage", "deb", "rpm", "snap"],
```

I'm not sure how popular this format is, but it was easy to download & run to app.